yaml-tests/src/test/java/YamlIntegrationDisabledPlannerRewritesTests.java

```diff
@@ -0,0 +1,265 @@
+/*
+ * YamlIntegrationTests.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2021-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.apple.foundationdb.relational.yamltests.MaintainYamlTestConfig;
+import com.apple.foundationdb.relational.yamltests.YamlTest;
+import com.apple.foundationdb.relational.yamltests.YamlTestConfigFilters;
+import org.junit.jupiter.api.TestTemplate;
+
+/**
+ * Class covering the standard integration tests specified by yamsql files.
```

Comment:
It may be good to note here that these are essentially all copies of the tests in `YamlIntegrationTests` but with the planner rewrites disabled in order to test that the rewriter can be safely flipped on and off as we roll it out. (And probably noting that this file is supposed to be temporary, as we don't think we'll need it once the rewriter is more mature.)